ui/app/mainui/AppMain.qml

```diff
@@ -353,6 +350,11 @@ Item {
                 receivedAmount = currencyStore.formatCurrencyAmountFromBigInt(toAmount, toToken.symbol, toToken.decimals)
             }
 
+            if (!!txHash) {
+                toastLink = "%1/tx/%2".arg(blockExplorerUrl).arg(txHash)
```

Comment:
As I see `blockExplorerURL` for all (almost all) chains ends with `/`, maybe we can strip out the second `/` before `tx` if it appears in the link?

Reply:
I think best we keep it instead of removing it in case that its present, it works with `double /` anyways.